code/game/supplyshuttle.dm

```diff
@@ -46,6 +46,192 @@ var/datum/controller/supply/supply_controller = new()
 
```

Comment:
all docking port classes should be moved to where the rest of the shuttles code is - for discoverability purposes, all shuttles should be in the same part of the code base